translator/isValid.go

```diff
@@ -11,7 +11,9 @@ import (
 //ErrorMessages will provide detail error messages to user
 var ErrorMessages = []string{}
 var InfoMessages = []string{}
-var ValidRetentionInDays = []string{"-1", "1", "3", "5", "7", "14", "30", "60", "90", "120", "150", "180", "365", "400", "545", "731", "1827", "3653"}
+
+// ValidRetentionInDays is based on what's supported by PutRetentionPolicy. See https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#CWL-PutRetentionPolicy-request-retentionInDays
```

Comment:
Just as an aside, we should also note that -1 is not a valid retention value. It's something that we use in the agent to denote no retention policy. Is this called out on public docs for the agent configuration separately?

Reply:
Im onboard with writing an integration test I don't think this is something i'll be able to add this sprint but retention in days as -1 is called out in the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html#CloudWatch-Agent-Configuration-File-Logssection).

> If you specify -1 or omit this field for a log group that the agent is creating now, the retention will be set to never expire. If the agent is sending logs to a log group that already exists, and you omit this field, the log group's retention is not changed.

But I do think we should edit the second sentence to say:
  If the agent is sending logs to a log group that already exists, and you omit this field [or set it to -1], the log group's retention is not changed.

Reply:
I think that linking to https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html#CloudWatch-Agent-Configuration-File-Logssection is the better thing to point to in documentation here.